x-pack/test/api_integration/apis/uptime/graphql/fixtures/snapshot_filtered_by_down.json

```diff
@@ -1,5 +1,5 @@
 {
   "snapshot": {
-    "counts": { "down": 2, "mixed": 0, "up": 0, "total": 2 }
+    "counts": { "down": 2, "mixed": 0, "up": 0, "total": 10 }
```

Comment:
I like that we're doing this now. I think. 😺

Reply:
I would categorize the previous behavior as a bug IMHO.